### PR TITLE
Aesthetic spec tweaks

### PR DESCRIPTION
**Pull request by hadley**
A few small improvements to the aesthetic specifications vignette

### CODE REVIEW

File: vignettes/ggplot2-specs.Rmd [hunk 1]
```diff
@@ -234,6 +232,9 @@ ggplot(df, aes(1, x)) +
   geom_text(aes(label = fontface, fontface = fontface))
 ```
 
+### Font size
+
+The `size` of text is measured in mm. 1 pt = 0.35 mm.
```
Comment (clauswilke) [inline vignettes/ggplot2-specs.Rmd]: Maybe add one additional sentence explaining what this means in practice? Something like: "1 pt = 0.35 mm, and ggplot2 provides this conversion factor in the variable `.pt`, which is defined as the number of points corresponding to 1 mm. Therefore, if you want to draw text in 12pt font, set `size = 12/.pt`."

**Comment by thomasp85** [comment]
@Hadley are you content with this or do you want to take Claus' suggestions Ito account?

**Comment by hadley** [comment]
Done.

File: vignettes/ggplot2-specs.Rmd [hunk 2]
```diff
@@ -234,7 +234,7 @@ ggplot(df, aes(1, x)) +
 
 ### Font size
 
-The `size` of text is measured in mm. 1 pt = 0.35 mm.
+The `size` of text is measured in mm. This is unusual, but makes the size of text consistent with the size of lines and points. Typically you specify font size using points (or pt for short), where 1 pt = 0.35. ggplot2 provides this conversion factor in the variable `.pt`, so if you want to draw 12pt text, set `size = 12 / .pt`.
```
Comment (clauswilke) [inline vignettes/ggplot2-specs.Rmd]: Is this missing "mm"?

> where 1 pt = 0.35.

Comment (hadley) [reply vignettes/ggplot2-specs.Rmd]: Oops, fixed.

**Pull request merged**